File: backend/components/metiers/threed/security/docs/samples/rgpd/samples/sample_rgpd.js
// sample_rgpd.js
// Exemple d’intégration avancée pour la conformité RGPD

const { anonymizeData, checkConsent } = require('./helpers_rgpd');

const user = { id: 'u123', name: 'Alice', consent: true };
console.log('Consent:', checkConsent(user));
console.log('Anonymized:', anonymizeData(user));

File: backend/components/metiers/threed/security/docs/samples/rgpd/samples/sample_rgpd.py
# sample_rgpd.py
# Exemple d’intégration avancée pour la conformité RGPD

from helpers_rgpd import anonymize_data, check_consent

user = {'id': 'u123', 'name': 'Alice', 'consent': True}
print('Consent:', check_consent(user))
print('Anonymized:', anonymize_data(user))
